### PR TITLE
[YUNIKORN-123] release v0.8.0

### PR DESCRIPTION
**Pull request by wilfred-s**
download link and release page added

### CODE REVIEW

File: release/v0.8.0.html [hunk 1]
```diff
@@ -121,22 +121,47 @@
...
+                    <p>In YuniKorn, there are lots of optimizations to improve the performance, such as a fully async event-driven system and low-latency sorting policies.</p>
+                    <p>Based on our test using Kubemark, YuniKorn is 2x+ faster than the default K8s scheduler.</p>
+
+                    <h2>Community</h2>
+                    <p>Since the birth of the YuniKorn project, the feedback has been well-received. We are thankful to have such a diverse community where we have members from Microsoft, LinkedIn, Alibaba, Apple, Tencent, Nvidia, and Cloudera, to help and guide the project’s growth.</p>
+                    <p>Many thanks to everyone who contributed to the release, and everyone in the Apache YuniKorn (Incubating) community!</p>
+                    <p>The release is a result of direct and indirect efforts from many contributors, listed below are those who contributed directly by submitting patches: Adam Antal, Akhil Puthenveettil Balan, Bhuvan Mysore, Jetly Jaimin, Kinga Marton, Sunil Govindan, Tao Yang,  Tarun Parimi, Wangda Tan, Wanqiang Ji, Weiwei Yang, Wilfred Spiegelenburg, Vinod Kumar Vavilapalli.</p>
```
Comment (sunilgovind) [inline release/v0.8.0.html:149]: typo. The -> This release

File: release/v0.8.0.html [hunk 2]
```diff
@@ -121,22 +121,47 @@
         <div class="row">
             <div class="col-md-12">
                 <h1>Apache YuniKorn (incubating) Release Notes v0.8.0</h1>
```
Comment (sunilgovind) [inline release/v0.8.0.html:123]: incubating  -> Incubating

Comment (wilfred-s) [reply release/v0.8.0.html:123]: fixed in all locations on the site

File: release/v0.8.0.html (hunk 1 above)
Comment (wilfred-s) [reply release/v0.8.0.html:149]: fixed

File: download.html [hunk 3]
```diff
@@ -135,19 +135,15 @@ <h1>Download Apache YuniKorn (incubating)</h1>
...
-                    <td>
-                        <!--
-                        <a href="https://www.apache.org/dyn/closer.cgi/incubator/yunikorn/v0.8.0/apache-yunikorn-0.8.0-incubating-src.tar.gz">source</a>
-                        (<a href="https://downloads.apache.org/incubator/yunikorn/v0.8.0/apache-yunikorn-0.8.0-incubating-src.tar.gz.sha512">checksum</a>
-                        <a href="https://downloads.apache.org/incubator/yunikorn/v0.8.0/apache-yunikorn-0.8.0-incubating-src.asc">signature</a>)
-                        -->
+                    <td style="vertical-align: middle;">v0.8.0</td>
+                    <td style="vertical-align: middle;">2020-05-04</td>
+                    <td style="vertical-align: middle;">
+                        <a href="https://www.apache.org/dyn/closer.cgi/incubator/yunikorn/0.8.0-incubating/apache-yunikorn-0.8.0-incubating-src.tar.gz" class="btn btn-success btn-sm active" role="button">Download</a>
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply download.html:141]: changed, I based the look on what we had in the hadoop site.

**Comment by wilfred-s** [comment]
changes pushed to master, updating site structure based on this

**Pull request closed without merging**